Stratis.Bitcoin/Features/MemoryPool/MempoolManager.cs

```diff
@@ -9,23 +9,49 @@
 
 namespace Stratis.Bitcoin.Features.MemoryPool
 {
+    /// <summary>
+    /// Memory pool scheduler.
+    /// </summary>
     public class MempoolScheduler : AsyncLock
```

Comment:
I guess sits up to the dev, I like to have smaller connected classes in the same file, but I suspect with time people will pull every thing to its own class.